### ISSUE TITLE
Importing synonyms

### ISSUE THREAD

**Issue by cmrn-rhi** [FoodOntology/foodon#280]
**Discussion for the FoodOn Curation Group: Should we be importing all synonyms from other ontologies?**
Sometimes it seems like we import synonym that goes to a level of detail greater than needed for FoodOn's operations, giving FoodOn unnecessary bloat. For example, CHEBI chemical terms often go into very descriptive synonym (e.g. broad and narrow synonyms) that FoodOn doesn't need, but the taxonomic synonymy of NCBITaxon would still be useful for organism organization. People can still look up the synonymies in other ontologies, just not FoodOn.
**TLDR:** Should we make it the default to keep importing exact synonymy, but then otherwise exclude others (broad, narrow, related, etc.) unless the domain ontology is key to FoodOn applications?

**Comment by ddooley** [comment]
See also: https://docs.google.com/document/d/1TJURVYMbAqj15GMnO1G3vP0Shf1tD-014eJrMImfhfw/

**Comment by ddooley** [comment]
Import of ChEBI annotation terms has been minimized. Closing this as its not so much an issue as a policy.
